### ISSUE TITLE
Devel::CheckOS:  isn't a legal OS name (CentOS, Fedora)

### ISSUE THREAD

**Issue by eserte** [DrHyde/perl-modules-Devel-CheckOS#26]
On my CentOS and Fedora systems the test suite fails:
```
Devel::CheckOS:  isn't a legal OS name
t/case-insensitive.t ............ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by DrHyde** [comment]
Curses! I know exactly what the problem is, that's because I have specific modules for Debian and Ubuntu, on which I tested, but other Linux distros are just detected as generic Linux, and t/case-insensitive.t discards that because it's an OS _family_. Easy fix that I'll get out tonight. Thanks!

**Comment by DrHyde** [comment]
I don't have a Fedora to test on, but I think 7ae6af57 should fix that. Do you think you could test?

**Comment by DrHyde** [comment]
Fixed (and tested on Arch; 1.90 fails there foo) in 1.91 which is on its way to the CPAN.
